src/qibolab/platforms/multiqubit.py

```diff
@@ -66,6 +67,15 @@ def set_gain(self, qubit, gain):
     def set_current(self, qubit, current):
         self.qb_port[qubit].current = current
 
+    def get_attenuation(self, qubit):
+        return self.ro_port[qubit].attenuation
+
+    def get_current(self, qubit):  # pragma: no cover
```

Comment:
Why do you add `# pragma: no cover` here?

Reply:
I think that it is a copy and paste error. Fixed.